Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is the second line.
```

Comment:
Done.

---
In reply to: [63123512](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r63123512) [](ancestors = 63123512)